src/mpyl/steps/deploy/k8s/__init__.py

```diff
@@ -23,6 +26,39 @@ def get_namespace(run_properties: RunProperties, project: Project) -> Optional[s
     return get_namespace_from_project(project)
 
 
+def rollout_restart_deployment(
+    logger: Logger, namespace: str, deployment: str
+) -> Output:
+    # from https://stackoverflow.com/a/67491253
+    v1_apps = client.AppsV1Api()
+
+    now = datetime.datetime.utcnow()
+    now_str = now.isoformat("T") + "Z"
+    body = {
+        "spec": {
+            "template": {
+                "metadata": {
+                    "annotations": {"kubectl.kubernetes.io/restartedAt": now_str}
+                }
+            }
+        }
+    }
+    try:
+        logger.info(f"Starting rollout restart of {deployment}...")
+        _, status_code, _ = v1_apps.patch_namespaced_deployment_with_http_info(
+            deployment, namespace, body, pretty="true"
+        )
+        msg = f"Rollout restart of {deployment} finished with statuscode {status_code}"
```

Comment:
This success message will never be used right, since we only return on failure and otherwise we always end up returning `deploy_result` (in `DeployDagster.execute`)? Same for the `replace_config_map` function below

Reply:
hmmm youre absolutely right. What do you think is the better approach. Moving the logging of the success into the `DagsterDeploy` step or flattening all success-Outputs into one Output that `DagsterDeploy` is returning?

Reply:
Agreed, let's put some logging and flatten the outputs

Reply:
done in 
65d5d77

Reply:
and logging taken care of in 
2449da9